### ISSUE TITLE
adding more environments

### ISSUE THREAD

**Issue by PolGuixe** [zetachang/react-native-dotenv#19]
Is there a way to add more environments e.g. .env.staging. ?

**Comment by mxmzb** [comment]
Yes that would be great. So we can have different env configs for simulator testing or device testing :)
